strongbox-cron-tasks/src/main/java/org.carlspring.strongbox/cron/config/ApplicationContextProvider.java

```diff
@@ -0,0 +1,29 @@
+package org.carlspring.strongbox.cron.config;
+
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Kate Novik.
+ */
+@Component
+public class ApplicationContextProvider implements ApplicationContextAware
```

Comment:
I don't think we need this at all. @AlexOreshkevich , could you please show @kate-novik how the rest of our Spring contexts are being set up across the rest of the project?